datafusion/src/sql/planner.rs

```diff
@@ -1256,7 +1256,7 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
                     BinaryOperator::Minus => Ok(Operator::Minus),
                     BinaryOperator::Multiply => Ok(Operator::Multiply),
                     BinaryOperator::Divide => Ok(Operator::Divide),
-                    BinaryOperator::Modulus => Ok(Operator::Modulus),
+                    BinaryOperator::Modulo => Ok(Operator::Modulo),
```

Comment:
`sqlparser` renamed this to Modulo as that is a more correct name for the remainder function. I did the same for DataFusion, though we can leave `Operator::Modulus` if that is desired

Reply:
Seems to me like it is worth fixing everywhere consistently